7_react/react1/week3/vite-project/src/components/AddTodo.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+
+function AddTodo({addTodo}){
```

Comment:
Great idea to extract this into its own component with its own state! 👍 Perhaps the validation logic should reside in here as well? I lost my changes when my input failed validation and had to type it again. 🥺 

You can also leverage the browser to do some of the validation! If you put the `required` attribute onto `<input>`s, the browser will tell the user to fill the fields out before submitting the form. 😄